frontend/src/utils/helpers/setMenuToggle.tsx

```diff
@@ -0,0 +1,16 @@
+import Cookies from 'js-cookie';
+import { MENU_VALUES } from 'constants/menuPaths';
+
+export const setMenuToggle = (): boolean | undefined => {
+  const menuStatus: string | undefined = Cookies.get(MENU_VALUES.menuStatus);
+  if (menuStatus === undefined) {
+    Cookies.set(MENU_VALUES.menuStatus, 'false');
+    return false;
+  }
```

Comment:
Do we need to save it to cookies? Why can't we just store it in state in MenuContainer with useState?

Reply:
Wep, bacause useState skied each time when we rebuild viretual-dom. As result we will have default menu-tab state if we update page/ switch page first time. For example I opened `help page` and hide menu(only icons show) and if i will click on other tab i will make rerender and get default state - my menu tabs will be open again. Its happens cause useState "cancels" when virtual-dom updates.